extensions/amp-analytics/0.1/activity-impl.js

```diff
@@ -119,6 +123,15 @@ const ACTIVE_EVENT_TYPES = [
   'mousedown', 'mouseup', 'mousemove', 'keydown', 'keyup',
 ];
 
+/**
+ * @param {!../../../src/service/ampdoc-impl.AmpDoc} ampDoc
+ * @return {Activity}
+ */
+export function activityServiceForTesting(ampDoc) {
```

Comment:
I'm going to have to dig into this some more tomorrow.

I'm not on hangouts today as I'm snowed in without a company device. 😞